src/test/regress/spec/isolation_citus_pause_node.spec

```diff
@@ -0,0 +1,171 @@
+setup
+{
+	SET citus.shard_replication_factor to 1;
+
+    CREATE TABLE company(id int primary key, name text);
+	select create_distributed_table('company', 'id');
+
+	create table employee(id int , name text, company_id int  );
```

Comment:
Can you also add a reference table, Citus local table, and a tenant table?

Reply:
Added a reference table. company is distributed table already. How can I add a local table into a worker node? In my case I need to add a local tavle into the worker node with id 3

Reply:
I gave info on Citus local tables above. You can try to pause coordinator after adding local table (its oly placement will be on coordinator). Then, a concurrent session should be blocked on any DML.